### PR TITLE
Removes the multitool mechcomp popup menu from airlocks if the panel is open

### PR DESCRIPTION
**Pull request by TTerc**
[qol]

## About the PR 

Title

## Why's this needed? 

From my understanding, it's a side effect of some recent mechcomp changes, which made hacking doors a bit annoying, since you need to close the popup window each time when clicking the door with a multitool.

### CODE REVIEW

File: code/datums/components/mechComp_signals.dm [hunk 1]
```diff
@@ -297,7 +297,8 @@
 
 //If it's a multi-tool, let the user configure the device.
 /datum/component/mechanics_holder/proc/attackby(var/comsig_target, obj/item/W as obj, mob/user)
-	if(!ispulsingtool(W) || !isliving(user) || user.stat)
+	var/obj/machinery/door/hacked_door = comsig_target
```
Comment (ZeWaka) [inline code/datums/components/mechComp_signals.dm:300]: It's always a door?

Comment (ZeWaka) [reply code/datums/components/mechComp_signals.dm:300]: It always exists and is passed in?

Comment (TTerc) [reply code/datums/components/mechComp_signals.dm:300]: comsig_target is the mechcomp component, could be any component
it's a workaround for the case when the component is a door with an open panel
can't really make it work via istype due to how the whole proc is coded so I did it that way :iiam:

Comment (TTerc) [reply code/datums/components/mechComp_signals.dm]: yea, I realized what I did now, oops
now the windows works for things other than doors aswell

**Pull request merged**